OracleIdentityGovernance/samples/containerizedDB/docker-compose.yaml

```diff
@@ -19,16 +19,17 @@ services:
   # The Oracle DB Definition
   #
   oimdb:
-    image: ${DC_REGISTRY_DB}/oracle/database:12.2.0.1-ee
+    image: ${DC_REGISTRY_DB}/oracle/database:latest
```

Comment:
Again, I recommend against using `latest` as when that changes, this image may no longer work (or vice-versa).